### PR TITLE
release 3.1

### LINKED ISSUE #278: loadAcquisition wizard broken during import

### PR DESCRIPTION
**Pull request by azazellochg**
Hi @delarosatrevin , I think we can rid of 3.0 code. Relion 4.0 is in alpha already, will be in beta this year. 
- remove all 3.0 code, except convert
- fix #278, #274, #277, #257
- add test to read old star format
- fix shifts conversion for pre-3.1 (origins are in px already)
- fix accuracy label, use Angstroms by default
- add a warning about polishing gain bug
- add EER support for motion correction
- refactor mc viewer
- replace xmipp by eman2 for psd calculation
- add 3.1.2 binary

### CODE REVIEW

**Comment by azazellochg** [comment]
Import / convert old star files to be checked

**Comment by azazellochg** [comment]
@delarosatrevin https://github.com/scipion-em/scipion-em-relion/pull/272/commits/4c3e3c0a4f63a5fe63a484f2b7f00244e204fb24 requires your closer look :)

**Comment by azazellochg** [comment]
EER needs more testing and some discussion w @pconesa and @delarosatrevin

**Comment by pconesa** [comment]
> EER needs more testing and some discussion w @pconesa and @delarosatrevin

Let me know how can I help.

File: setup.py [hunk 1]
```diff
@@ -165,7 +165,7 @@
     # MANIFEST.in as well.
     # include_package_data=True,
     package_data={  # Optional
-       'relion': ['relion_logo.png', 'protocols.conf', 'convert/mtfs/*.star'],
```
Comment (delarosatrevin) [inline setup.py:168]: Relion logo file is still a png file.

**Review by delarosatrevin — CHANGES_REQUESTED** [review]
All tests (except MultiBody) are passing. Just a minor comment

File: setup.py (hunk 1 above)
Comment (azazellochg) [reply setup.py:168]: I have uploaded a new jpg logo

**Comment by azazellochg** [comment]
I'm merging this as no comments were raised.

**Pull request merged**